msal/changelog.txt

```diff
@@ -1,3 +1,14 @@
+2.0.1-preview
+=============
+This release contains:
+- Resolved logger issue in B2C pm sample for msal with object null reference.
```

Comment:
I would remove this.  Henrik made a fix because netcore2 apps were not loading netcore1, and we got the exception in the logger. but all this is internal. I don't think we need to mention it here.